viewer/packages/rendering/src/glsl/post-processing/ssao-blur-combine.frag

```diff
@@ -1,7 +1,5 @@
-// Copyright Cognite (C) 2019 Cognite
+// Copyright Cognite (C) 2021 Cognite
 //
-// Efficient Gaussian blur based on technique described by Daniel Rákos in
-// http://rastergrid.com/blog/2010/09/efficient-gaussian-blur-with-linear-sampling/
```

Comment:
I think we should leave the reference, as it is still kinda based on this. And is also a good resource for understanding the shader.